oeps/oep-0018-bp-python-dependencies.rst

```diff
@@ -234,17 +234,22 @@ Some guidelines to keep in mind when adding constraints:
   constraint has been imposed.  If there is an issue (either in Jira or an
   upstream issue tracker) for resolving the problem, a link to it should be
   included in the comment.
-* Minimum versions are generally not necessary for IDAs since ``pip-compile``
-  always tries to use the latest compatible version in the generated
-  requirements files; if ``make upgrade`` produces a downgrade in the compiled
-  dependencies, it should be carefully investigated before committing.
-  For library packages, on the other hand, dependency resolution is performed
-  outside of the library's codebase, and specifying minimum version constraints
-  in ``base.in`` when the code depends on newer features is the only way to
-  ensure that a version conflict is detected when the IDA's dependency
-  resolution occurs.
-
-These constraints should be periodically reviewed to determine if some of them
+* Minimum constraints are required in some circumstances, and should be avoided
+  in others.
```

Comment:
FYI: Not sure if you prefer consistency with the rest of the doc, but there should be no hard wraps.

Reply:
Yeah, I was keeping it for consistency. We could do a pass over the repo at some point to unwrap lines.